### PR TITLE
Modify_Amplify_resources

### PR DESCRIPTION
**Pull request by AnilMaktala**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

**Review by AnilMaktala — COMMENTED** [review]
updated

**Review by renebrandel — APPROVED** [review]

File: src/pages/gen2/build-a-backend/data/data-modeling/identifiers/index.mdx [hunk 1]
```diff
@@ -45,11 +45,12 @@ const schema = a.schema({
     title: a.string(),
     completed: a.boolean(),
   }).identifier(['todoId'])
+  .authorization([a.allow.public()]),
 });
 
 const client = generateClient<Schema>();
 
-const todo = await client.models.Todo.create({ todoId: 'MyUniqueTodoId', title: 'Buy Milk', completed: false });
+ const { data: todo, errors } = await client.models.Todo.create({ todoId: 'MyUniqueTodoId', title: 'Buy Milk', completed: false });
```
Comment (ykethan) [inline src/pages/gen2/build-a-backend/data/data-modeling/identifiers/index.mdx:53]: do we want to keep the unused variable `errors` in the example?

Comment (renebrandel) [reply src/pages/gen2/build-a-backend/data/data-modeling/identifiers/index.mdx:53]: yeah, just for visibility that it's available.

**Review by chrisbonifacio — APPROVED** [review]
LGTM!

**Review by renebrandel — APPROVED** [review]

**Review by chrisbonifacio — APPROVED** [review]

**Pull request merged**